en-US/Docs/ReleaseNotesRTM.md

```diff
@@ -4,4 +4,4 @@ title: Release Notes
 permalink: /en-US/Docs/ReleaseNotesRTM.htm
 lang: en-US
 ---
-{% include redirect.html url="https://docs.microsoft.com/en-us/windows/iot-core/release-notes/commercial/creatorsupdate" %}
```

Comment:
This looks wrong

Reply:
It should be this: https://docs.microsoft.com/en-us/windows/iot-core/release-notes/commercial/

Reply:
This is how Sara wanted the document to look on docs.microsoft.

Reply:
We'll want it to land on https://docs.microsoft.com/en-us/windows/iot-core/release-notes/commercial/fallcreatorsupdate once the PR in docs is merged.